### ISSUE TITLE
EOS node periodically start delaying from mainnet, restart needed

### ISSUE THREAD

**Issue by chukynax** [EOSIO/eos#8154]
running node-eos dockerized appliance, from time to time node starts delaying from main network, no errors in the console, all peers work.

**Comment by heifner** [comment]
What version? 
What are your specs? Are you running on SSD? How powerful is your machine?

**Comment by chukynax** [comment]
nodeos version v1.8.5
non ssd
10 gb ram
CPU usage is almost nothing


```
    # Replace list with fresh from https://eosnodes.privex.io/?config=1
    # updated 2019-11-1
    p2p-peer-address = api-full1.eoseoul.io:9876
    p2p-peer-address = api-full2.eoseoul.io:9876
    p2p-peer-address = boot.eostitan.com:9876
    p2p-peer-address = bp.cryptolions.io:9876
    p2p-peer-address = bp.eosbeijing.one:8080
    p2p-peer-address = bp.libertyblock.io:9800
    p2p-peer-address = br.eosrio.io:9876
    p2p-peer-address = eos-seed-de.privex.io:9876
    p2p-peer-address = eu1.eosdac.io:49876
    p2p-peer-address = fullnode.eoslaomao.com:443
    p2p-peer-address = mainnet.eoscalgary.io:5222
    p2p-peer-address = node.eosflare.io:1883
    p2p-peer-address = node1.eoscannon.io:59876
    p2p-peer-address = node1.eosnewyork.io:6987
    p2p-peer-address = node2.eosnewyork.io:6987
    p2p-peer-address = p.jeda.one:3322
    p2p-peer-address = p2p.eosdetroit.io:3018
    p2p-peer-address = p2p.genereos.io:9876
    p2p-peer-address = p2p.meet.one:9876
    p2p-peer-address = peer.eosn.io:9876
    p2p-peer-address = peer.main.alohaeos.com:9876
    p2p-peer-address = peer1.mainnet.helloeos.com.cn:80
    p2p-peer-address = peer2.mainnet.helloeos.com.cn:80
    p2p-peer-address = peering.mainnet.eoscanada.com:9876
    p2p-peer-address = peering1.mainnet.eosasia.one:80
    p2p-peer-address = peering2.mainnet.eosasia.one:80
    p2p-peer-address = publicnode.cypherglass.com:9876

    plugin = eosio::state_history_plugin
    plugin = eosio::chain_plugin
    plugin = eosio::chain_api_plugin
    plugin = eosio::net_api_plugin

    state-history-endpoint = 0.0.0.0:8080
    trace-history = true
    agent-name = EOS Agent
    chain-state-db-size-mb = 65536
    reversible-blocks-db-size-mb = 2048
    http-server-address = 0.0.0.0:8888
    p2p-listen-endpoint = 127.0.0.1:36780
    http-validate-host = false
    verbose-http-errors = true
    abi-serializer-max-time-ms = 2000
    chain-threads = 8
    access-control-allow-origin = *
    wasm-runtime = wabt
    last-block-time-offset-us = -300000
    max-clients = 150
    sync-fetch-span = 2000

    
    # blacklists
    #
    actor-blacklist = blacklistmee

    #https://eoscorearbitration.io/wp-content/uploads/2018/07/ECAF_Arbitrator_Order_2018-06-19-AO-001.pdf
    actor-blacklist = ge2dmmrqgene
    actor-blacklist = gu2timbsguge
    actor-blacklist = ge4tsmzvgege
    actor-blacklist = gezdonzygage
    actor-blacklist = ha4tkobrgqge
    actor-blacklist = ha4tamjtguge
    actor-blacklist = gq4dkmzzhege

    #https://eoscorearbitration.io/wp-content/uploads/2018/07/ECAF_Arbitrator_Order_2018-06-22-AO-002.pdf
    actor-blacklist = gu2teobyg4ge
    actor-blacklist = gq4demryhage
    actor-blacklist = q4dfv32fxfkx
    actor-blacklist = ktl2qk5h4bor
    actor-blacklist = haydqnbtgene
    actor-blacklist = g44dsojygyge
    actor-blacklist = guzdonzugmge
    actor-blacklist = ha4doojzgyge
    actor-blacklist = gu4damztgyge
    actor-blacklist = haytanjtgige
    actor-blacklist = exchangegdax
    actor-blacklist = cmod44jlp14k
    actor-blacklist = 2fxfvlvkil4e
    actor-blacklist = yxbdknr3hcxt
    actor-blacklist = yqjltendhyjp
    actor-blacklist = pm241porzybu
    actor-blacklist = xkc2gnxfiswe
    actor-blacklist = ic433gs42nky
    actor-blacklist = fueaji11lhzg
    actor-blacklist = w1ewnn4xufob
    actor-blacklist = ugunxsrux2a3
    actor-blacklist = gz3q24tq3r21
    actor-blacklist = u5rlltjtjoeo
    actor-blacklist = k5thoceysinj
    actor-blacklist = ebhck31fnxbi
    actor-blacklist = pvxbvdkces1x
    actor-blacklist = oucjrjjvkrom

    #https://eoscorearbitration.io/wp-content/uploads/2018/07/ECAF-Temporary-Freeze-Order-2018-07-13-AO-003.pdf
    actor-blacklist = neverlandwal
    actor-blacklist = tseol5n52kmo
    actor-blacklist = potus1111111

    #https://eoscorearbitration.io/wp-content/uploads/2018/07/ECAF-Order-of-Emergency-Protection-2018-07-19-AO-004.pdf
    actor-blacklist = craigspys211

    #https://eoscorearbitration.io/wp-content/uploads/2018/08/ECAF-Order-of-Emergency-Protection-2018-08-07-AO-005.pdf
    actor-blacklist = eosfomoplay1

    #https://eoscorearbitration.io/wp-content/uploads/2018/08/ECAF-Order-of-Emergency-Protection-2018-08-28-AO-006.pdf
    actor-blacklist = wangfuhuahua

    #https://eoscorearbitration.io/wp-content/uploads/2018/09/ECAF-Order-of-Emergency-Protection-2018-09-07-AO-008.pdf
    #https://eoscorearbitration.io/wp-content/uploads/2018/09/ECAF-Order-of-Emergency-Protection-2018-09-24-AO-010.pdf
    #actor-blacklist = ha4timrzguge
    actor-blacklist = guytqmbuhege

    #https://eoscorearbitration.io/wp-content/uploads/2018/09/ECAF-Order-of-Emergency-Protection-2018-09-09-AO-009.pdf
    actor-blacklist = huobldeposit

    #https://eoscorearbitration.io/wp-content/uploads/2018/09/ECAF-Order-of-Emergency-Protection-2018-09-25-AO-011.pdf
    actor-blacklist = gm3dcnqgenes
    actor-blacklist = gm34qnqrepqt
    actor-blacklist = gt3ftnqrrpqp
    actor-blacklist = gtwvtqptrpqp
    actor-blacklist = gm31qndrspqr
    actor-blacklist = lxl2atucpyos

    #https://eoscorearbitration.io/wp-content/uploads/2018/10/ECAF-Order-of-Emergency-Protection-2018-10-05-AO-012.pdf
    actor-blacklist = g4ytenbxgqge
    actor-blacklist = jinwen121212
    actor-blacklist = ha4tomztgage
    actor-blacklist = my1steosobag
    actor-blacklist = iloveyouplay
    actor-blacklist = eoschinaeos2
    actor-blacklist = eosholderkev
    actor-blacklist = dreams12true
    actor-blacklist = imarichman55

    #https://eoscorearbitration.io/wp-content/uploads/2018/10/ECAF-Order-of-Emergency-Protection-2018-10-05-AO-013.pdf
    actor-blacklist = gizdcnjyg4ge

    #https://eoscorearbitration.io/wp-content/uploads/2018/10/ECAF-Order-of-Emergency-Protection-2018-10-12-AO-014.pdf
    actor-blacklist = gyzdmmjsgige

    #https://eoscorearbitration.io/wp-content/uploads/2018/10/ECAF-Order-of-Emergency-Protection-2018-10-13-AO-015.pdf
    actor-blacklist = guzdanrugene
    actor-blacklist = earthsop1sys

    #https://eoscorearbitration.io/wp-content/uploads/2018/10/ECAF-Order-of-Emergency-Protection-2018-10-31-AO-017.pdf
    actor-blacklist = refundwallet
    actor-blacklist = jhonnywalker
    actor-blacklist = alibabaioeos
    actor-blacklist = whitegroupes
    actor-blacklist = 24cryptoshop
    actor-blacklist = minedtradeos

    #Manual at 2019.02.22 (MainNet gropup)
    actor-blacklist = newdexmobapp
    actor-blacklist = ftsqfgjoscma
    actor-blacklist = hpbcc4k42nxy
    actor-blacklist = 3qyty1khhkhv
    actor-blacklist = xzr2fbvxwtgt
    actor-blacklist = myqdqdj4qbge
    actor-blacklist = shprzailrazt
    actor-blacklist = qkwrmqowelyu
    actor-blacklist = lhjuy3gdkpq4
    actor-blacklist = lmfsopxpr324
    actor-blacklist = lcxunh51a1gt
    actor-blacklist = geydddsfkk5e
    actor-blacklist = pnsdiia1pcuy
    actor-blacklist = kwmvzswquqpb
    actor-blacklist = guagddoefdqu

```

**Comment by heifner** [comment]
I see you are running state_history_plugin. Are you hitting that endpoint hard? What does your disk utilization look like?

**Comment by chukynax** [comment]
yes we use it a lot

okay, little correction there is 5 ssd drives in raid5 and it uses it on 97%

**Comment by pr0logas** [comment]
@chukynax Just a small question off topic, how much disk space full node + history_plugin is using right now?

**Comment by chukynax** [comment]
718G

**Comment by pr0logas** [comment]
> 718G

Thanks!

**Comment by chukynax** [comment]
in the experimental way i cleared that disk utilization is made by histool application most ...

**Comment by heifner** [comment]
Resolved by #8271 & #8156
